src/components/FooterWrapper.js

```diff
@@ -10,59 +10,75 @@ import * as Utils from 'src/libs/utils'
 
 
 const styles = {
-  item: { marginLeft: '2rem' }
+  item: { marginLeft: '2rem' },
+  footer: {
+    flex: 'none',
+    padding: '0 1rem',
+    backgroundColor: colors.secondary(),
+    color: 'white'
+  }
 }
 
 const buildTimestamp = new Date(parseInt(process.env.REACT_APP_BUILD_TIMESTAMP, 10))
 
 // If you change the layout here, make sure it's reflected in the pre-rendered version in public/index.html
-const FooterWrapper = ({ children }) => {
+const FooterWrapper = ({ children, alwaysShow }) => {
+  const [isExpanded, setIsExpanded] = useState(false)
```

Comment:
```suggestion
  const [isExpanded, setIsExpanded] = useState(alwaysShow)
```

Reply:
This would work, but it creates a coupling between the two values that makes it slightly more complex. I think the behavior is a little clearer in its current form.